processor/src/main/java/org/mapstruct/ap/internal/model/BeanMappingMethod.java

```diff
@@ -96,6 +96,7 @@ public class BeanMappingMethod extends NormalTypeMappingMethod {
     private final MethodReference finalizerMethod;
 
     private final MappingReferences mappingReferences;
+    private List<Type> typeParameters;
```

Comment:
Why can't we make this final? I also see that this is not in the `getImportTypes()`. How will it with with things such as `extends` and `super`?